tests/test_peak_detection.py

```diff
@@ -1359,19 +1363,347 @@ def test_noisy_data_B1(noisy_data_B1):
         4815,
         5340,
         5897,
-        6450,
+        6449,
         7040,
         7570,
         8091,
-        8616,
-        9155,
+        8617,
+        9154,
         9602,
         10134,
         10686,
         11233,
-        11798,
+        11797,
         12347,
         12851,
     ]
     assert np.array_equal(peak_indices, expected_peak_indices)
     assert np.array_equal(valley_indices, expected_valley_indices)
+
+
+def test__B6_data_causing_TwoValleysInARowError(MA20123123__2020_10_13_173812__B6):
+    pipeline, peak_and_valley_indices = MA20123123__2020_10_13_173812__B6
+    # plot and save results
+    filtered_data = pipeline.get_noise_filtered_gmr()
+    _plot_data(
+        peak_and_valley_indices,
+        filtered_data,
+        os.path.join(PATH_TO_PNGS, "new_MA20123123__2020_10_13_173812__B6.png"),
+        x_bounds=(65, 75),
+    )
+
+    peak_indices, valley_indices = peak_and_valley_indices
+    find_twitch_indices(peak_and_valley_indices, filtered_data)
+
+    expected_peak_indices = [
```

Comment:
use `#fmt: off/on` to stop `black` from turning these arrays into dozens of lines of code

https://black.readthedocs.io/en/stable/the_black_code_style.html

You can put these arrays all on a single line. There's no reason we need to be able to see individual numerical values at every position when casually scrolling through the code